gsoc/gsoc2016/ideas_juliaastro.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title:  "Ideas for Julia Astro"
+show_main: false
+ideas_team: JuliaAstro
+---
+
+### Image compression and efficient table reading in FITSIO.jl
+
+*Suggested Mentor(s):* [Kyle Barbary](http://github.com/kbarbary)
```

Comment:
I would need you to find another person as a backup mentor, normally it would default to the sub-org admin if you are not available (holidays/conference/...) but you are the sub-org admin :)